src/common/models/filter-clause/filter-clause.ts

```diff
@@ -75,7 +75,7 @@ export class FilterClause implements Instance<FilterClauseValue, FilterClauseJS>
 
   static evaluate(selection: Expression, now: Date, maxTime: Date, timezone: Timezone): TimeRange {
     if (!selection) return null;
-    var maxTimeMinuteTop = minute.shift(minute.floor(maxTime || now, timezone), timezone, 1);
+    var maxTimeMinuteTop = minute.ceil(maxTime || now, timezone);
```

Comment:
these aren't the same right? like shifting a floored value by 1 could return something different from ceiling it?